package.json

```diff
@@ -59,7 +59,7 @@
   "dependencies": {
     "@types/xml2js": "^0.4.14",
     "async": "3.2.5",
-    "axios": "1.7.2",
+    "axios": "1.7.4",
```

Comment:
Not sure why version pinning is used, but observing semver could avoid this update.

Reply:
usually dependabot is bumping up those versions, will need to check.